packages/terra-date-input/src/DateInput.jsx

```diff
@@ -654,20 +661,31 @@ class DateInput extends React.Component {
       dateValue = `${year}-${month}-${day}`;
     }
 
+    const format = (DateInputUtil.computedDisplayFormat(this.props.displayFormat, this.props.intl.locale) === 'month-day-year')
+      ? `(${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`
+      : `(${this.props.intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`;
+
     return (
-      <div
-        {...customProps}
-        className={dateInputClassNames}
-        ref={(element) => { this.dateInputContainer.current = element; if (refCallback) { refCallback(element); } }}
-      >
-        <input
-          // Create a hidden input for storing the name and value attributes to use when submitting the form.
-          // The data stored in the value attribute will be the visible date in the date input but formatted in YYYY-MM-DD format.
-          type="hidden"
-          name={name}
-          value={dateValue}
-        />
-        {this.formattedRender()}
+      <div>
+        <div
+          {...customProps}
+          className={dateInputClassNames}
+          ref={(element) => { this.dateInputContainer.current = element; if (refCallback) { refCallback(element); } }}
+        >
+          <input
+            // Create a hidden input for storing the name and value attributes to use when submitting the form.
+            // The data stored in the value attribute will be the visible date in the date input but formatted in YYYY-MM-DD format.
+            type="hidden"
+            name={name}
+            value={dateValue}
+          />
+          {this.formattedRender()}
+          { (help === undefined) && (
+          <div id="format" className={cx('format-text')} aria-label={`Date Format: ${format}`}>
```

Comment:
As is, we will duplicate the `format` id if more than one DateInput is mounted at the same time. I assume this results in some weirdness with the screen readers if the formats vary between instances? We should scope this id or make it dynamic on a per-instance basis.

Reply:
Added `uuidv4` here  https://github.com/cerner/terra-framework/pull/1230/commits/34514197b3cf2dec580bd2b2d8374d6809fc9d23